open-metadata-implementation/governance-servers/open-lineage-services/open-lineage-services-server/src/main/java/org/odpi/openmetadata/governanceservers/openlineage/services/GraphServices.java

```diff
@@ -39,38 +39,38 @@
     /**
      * Returns a lineage subgraph.
      *
-     * @param scope        The scope queried by the user: hostview, tableview, columnview.
-     * @param lineageQuery source-and-destination, end-to-end, ultimate-source, ultimate-destination, glossary.
      * @param graphName    main, buffer, mock, history.
+     * @param scope source-and-destination, end-to-end, ultimate-source, ultimate-destination, glossary.
+     * @param view        The view queried by the user: hostview, tableview, columnview.
      * @param guid         The guid of the node of which the lineage is queried from.
      * @return A subgraph containing all relevant paths, in graphSON format.
      */
-    public String queryLineage(String scope, String lineageQuery, String graphName, String guid) {
+    public String lineage(String graphName, String scope, String view, String guid) {
         String response = "";
 
-        scope = reformatArg(scope);
-        lineageQuery = reformatArg(lineageQuery);
         graphName = reformatArg(graphName);
+        scope = reformatArg(scope);
+        view = reformatArg(view);
```

Comment:
This should not be necessary. Is it removed in next PR?

Reply:
Can only be removed when the enums are refactored